generators/app/tasks/copyTemplateFiles.js

```diff
@@ -1,50 +1,42 @@
 const mkdirp = require('mkdirp');
 
 const {
+  FILES,
+  FILES_TO_DELETE,
   TEMPLATE_FILES,
-  LINTER_PATH,
-  LOCAL_STORAGE_FILE,
   FLOWCONFIG_PATH,
-  REDUX_COMPONENTS,
   CI_CONFIG_FILE,
   LINTER_IGNORE_PATH
 } = require('../constants');
 
-const { copyTpl, copy } = require('./utils');
+const { copyTpl, copy, copyEjsTpl, deleteFiles } = require('./utils');
 
 module.exports = function copyTemplateFiles() {
-  const bindedCopyTpl = copyTpl.bind(this);
   const bindedCopy = copy.bind(this);
+  const bindedCopyTpl = copyTpl.bind(this);
+  const bindedCopyEjsTpl = copyEjsTpl.bind(this);
+  const bindedDeleteFiles = deleteFiles.bind(this);
+
+  FILES_TO_DELETE.forEach(src => bindedDeleteFiles(src));
+
+  FILES.forEach(path => bindedCopy(path, path, null, { projectName: this.projectName }));
 
-  TEMPLATE_FILES.forEach(path => bindedCopy(path, path, null, { projectName: this.projectName }));
+  TEMPLATE_FILES.forEach(path => bindedCopyEjsTpl(path));
 
   mkdirp(this.destinationPath(`${this.projectName}/src/app/assets/`));
 
   if (this.features.flow) {
     bindedCopy(FLOWCONFIG_PATH.src, FLOWCONFIG_PATH.destination);
   }
 
-  if (this.features.redux) {
-    REDUX_COMPONENTS.forEach(path => bindedCopy(path, path, null, { projectName: this.projectName }));
-  }
-
-  bindedCopy(LINTER_PATH.src, LINTER_PATH.destination);
+  // Try to delete this
```

Comment:
should we leave this comment?

Reply:
I will change a little bit this file in next pr :D